### DISCUSSION TITLE
Q: How to use GraphQL to find only repositories owned by organizations?

### DISCUSSION THREAD

**Discussion by Ranicki69** [community/community#24519] [Apps, API and Webhooks]
<p>Does anyobody know if it is possible to query Github with GraphQL and only display repositories that are owned by organizations?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/ranicki69">@ranicki69</a>,</p>
<p>Thanks for being here! Something like this should work:</p>
<pre><code>{
  organization(login: "atom") {
      repositories(first: 50) {
        nodes {
          name
        }
        pageInfo {
          hasNextPage
        }
      }
    }
  }</code></pre>
